### ISSUE TITLE
[TF2] Start-up video not playing in x64 version

### ISSUE THREAD

**Issue by maksw2** [ValveSoftware/Source-1-Games#5895]
Hello, i'm experiencing a bug where tf.exe properly plays the start-up video but tf_win64.exe doesn't.

**Comment by Benioteniso** [comment]
i`m (kind of) having this error too, and looks like its related to valve changing the .bik (a old file format, known to be used in valve.bik Aka **Tf2** default intro) to .Webm.
changing the file to .Webm doesn`t help too

###  mods that modify the intro needs to change the format to .Webm
(Sorry if my english is bad)

**Comment by maksw2** [comment]
That's cool! I got the video working and still figuring out the audio. Should be mentioned somewhere..

**Comment by just-a-github-user** [comment]
Guessin you need to add startupvids.txt inside tf/media, and then add the string "media/valve.webm" inside the file, check if there's a valve.webm inside the tf/media folder, then it'll launch?

**Comment by maksw2** [comment]
That's right! It's actually mentioned in the documentation (WebM page) but nowhere else.

**Comment by maksw2** [comment]
Only Bink playes in 32 bit version, and only WebM playes in 64 bit version.

**Comment by maksw2** [comment]
That's not mentioned anywhere.

**Comment by megakarlach** [comment]
Also noticed same problem in CS:S, HL2:DM, and possibly DOD:S and HL DMS aswell. Valve just upgraded all these games to TF2 branch recently in Feb 18, 2025, and Valve forgot to add WebM version of Valve intro, causing it to skip when launched in 64-bit (but does play in 32-bit). This has been fixed in TF2 through.

For these other games, simply copying "valve.webm" from TF2's hl2\media and paste it to HL2: DM's hl2\media fixes the issue.

@kisak-valve - Add CS:S, HL2: DM, HL:DMS and DOD:S labels, and maybe change the title since those games are affected aswell.
